### PR TITLE
Extending the select component to support filtering

### PR DESCRIPTION
**Pull request by andresenra**
image

### CODE REVIEW

**Review by Fasani — APPROVED** [review]

**Comment by Fasani** [comment]
@andresenra this has a conflict now.

**Comment by Fasani** [comment]
@andresenra also it seems a rather larger number of lines being removed? Any idea?

**Comment by WiktorStarczewski** [comment]
I like it, quite invasive though so might not survive a 1.0 update - we'll need to implement it again there.
(1.0 is in rc1 now and progressing)

File: js/forms.js [hunk 1]
```diff
@@ -513,6 +513,45 @@
...
+                      $(option).parent().show();
+                  }
+              }
+          });
+      }
+
+      //Added to search
+      var setSearchableOption = function () {
+          var placeholder = $select.attr('searchable');
+          var element = $('<span><input type="text" class="search" style="margin: 5px 0px 16px 15px; width: 96%;" placeholder="' + placeholder + '"></span>');
```
Comment (WiktorStarczewski) [inline js/forms.js]: Why inline style and not a class?

**Review by WiktorStarczewski — CHANGES_REQUESTED** [review]

**Comment by Fasani** [comment]
Fixed scroll bug. Added support for max number of selected items.

https://www.useloom.com/share/66995ef79f8741c3a04d93e868654265

**Comment by Fasani** [comment]
@WiktorStarczewski and @andresenra please review and see video

**Review by WiktorStarczewski — APPROVED** [review]

**Pull request merged**